crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html

```diff
@@ -0,0 +1,21 @@
+{% load static %}
+
+<span>
+  <a href="#" id="printout_report" class="bulk-print-link">
+    <img src="{% static "img/intake-icons/ic_printer.svg" %}" alt="printer" class="icon" />
+    Print all
+  </a>
+</span>
+
+<div class="bulk-print-reports display-none">
+  {% for report in print_reports|slice:":100" %}
```

Comment:
Ooo, hard coding `100` here results in us rendering 100 reports on the page _anytime_ `print_reports >= 100`. 
With the current UI, a user can only actually print those 100 reports if they've selected -- passed 100 IDs in their `GET`, or `print_reports.count() == 100`.

There's an opportunity here to save a bit of work and limit what we're rendering in the template to reports which can be printed by the end user. I'd call this an optimization and we don't need to block merging this whole PR on it but wanted to mention!

Reply:
Good catch. I'm playing with this idea over at https://github.com/usdoj-crt/crt-portal/pull/739